rector_examples/path_alias_manager_updated.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * This demonstrates the deprecated static calls that might be called from procedural code like `.module` files.
+ */
+
+/**
+ * A simple example using the minimum number of arguments.
+ */
+function simple_example() {
+  \Drupal::service('path_alias.manager');
+}
```

Comment:
We would like to have another example file that shows how the code would be updated.